pkg/audit/manager.go

```diff
@@ -75,7 +75,7 @@ type Manager struct {
 	mgr             manager.Manager
 	ucloop          *updateConstraintLoop
 	reporter        *reporter
-	log             logr.Logger
+	logger          logr.Logger
```

Comment:
to minimize diffs, we should avoid renaming variables unless there is a tangible benefit.

Reply:
i think `logger` is a better name for our logger and the diff doesn't change that much since we replace all `log.` with `h.log` for the webhooks but I made the change.